src/lib/service/civilServantRegistry/csrsService.spec.ts

```diff
@@ -0,0 +1,76 @@
+import {expect} from 'chai'
+import * as sinon from 'sinon'
+
+import {OrganisationalUnit, User} from '../../model'
+import * as csrsService from './csrsService'
+import {OrganisationalUnitCache} from './organisationalUnit/organisationalUnitCache'
+import {OrganisationalUnitTypeaheadCache} from './organisationalUnit/organisationalUnitTypeaheadCache'
+import * as organisationalUnitClient from './organisationalUnit/organisationUnitClient'
+
+function getOrg(orgName: string, id: number, parentId?: number) {
+	const org = new OrganisationalUnit()
+	org.name = orgName
+	org.id = id
+	if (parentId) {
+		org.parentId = parentId
+	}
+	return org
+}
+
+describe('CsrsService tests', () => {
```

Comment:
Might be a good idea to write a few tests for `getOrganisation` covering the cases in your `if` statements (eg. `org === undefined`). Probably easier if you mock the response from `getOrganisationFromApi` and `organisationalUnitCache.get`.

Reply:
Resolved - copied the tests over from the lpg-management PR (the code is identical more or less)